packages/zwave-js/src/lib/commandclass/PowerlevelCC.ts

```diff
@@ -0,0 +1,409 @@
+import {
+	CommandClasses,
+	enumValuesToMetadataStates,
+	Maybe,
+	MessageOrCCLogEntry,
+	MessageRecord,
+	validatePayload,
+	ValueMetadata,
+	ZWaveError,
+	ZWaveErrorCodes,
+} from "@zwave-js/core";
+import { getEnumMemberName, pick } from "@zwave-js/shared";
+import type { Driver } from "../driver/Driver";
+import { NodeStatus } from "../node/Types";
+import { CCAPI } from "./API";
+import {
+	API,
+	CCCommand,
+	CCCommandOptions,
+	ccValue,
+	ccValueMetadata,
+	CommandClass,
+	commandClass,
+	CommandClassDeserializationOptions,
+	expectedCCResponse,
+	gotDeserializationOptions,
+	implementedVersion,
+} from "./CommandClass";
+
+// All the supported commands
+export enum PowerlevelCommand {
+	Set = 0x01,
+	Get = 0x02,
+	Report = 0x03,
+	TestNodeSet = 0x04,
+	TestNodeGet = 0x05,
+	TestNodeReport = 0x06,
+}
+
+export enum Powerlevel {
+	NormalPower = 0x00,
+	Minus1dBm = 0x01,
+	Minus2dBm = 0x02,
+	Minus3dBm = 0x03,
+	Minus4dBm = 0x04,
+	Minus5dBm = 0x05,
+	Minus6dBm = 0x06,
+	Minus7dBm = 0x07,
+	Minus8dBm = 0x08,
+	Minus9dBm = 0x09,
+}
```

Comment:
Since these will end up as labels somewhere through the metadata, I'm thinking they should be formatted like this:
```suggestion
export enum Powerlevel {
	"Normal Power" = 0x00,
	"-1 dBm" = 0x01,
	"-2 dBm" = 0x02,
	"-3 dBm" = 0x03,
	"-4 dBm" = 0x04,
	"-5 dBm" = 0x05,
	"-6 dBm" = 0x06,
	"-7 dBm" = 0x07,
	"-8 dBm" = 0x08,
	"-9 dBm" = 0x09,
}
```